pubspec.yaml

```diff
@@ -11,3 +11,6 @@ dev_dependencies:
   fake_async: "^0.1.2"
   stack_trace: "^1.0.0"
   test: "^0.12.0"
+  build_runner: ^0.7.11
```

Comment:
I'd add a `# note` here about why these are included – without other documentation or usage, it's not clear